pkgs/development/ocaml-modules/color/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchurl
+, buildDunePackage
+, gg
+}:
+
+buildDunePackage rec {
+  pname = "color";
+  version = "0.2.0";
+
+  useDune2 = true;
+
+  src = fetchurl {
+    url = "https://github.com/anuragsoni/${pname}/releases/download/${version}/color-${version}.tbz";
```

Comment:
Is there a reason we cannot fetch the git repository instead of the release archive? If so, please document it.

As an aside: I personally prefer not to use `pname` in the source definition since the package name and the repository name might not change together. This is rarely relevant in practice though, and it's a matter of opinion. You do not need to change it.